kde/kmymoney4-devel/Portfile

```diff
@@ -41,8 +40,7 @@ depends_lib-append  port:libofx \
 
 patch.dir         ${workpath}/${distname}
 patch.pre_args    -p1
-patchfiles        patch-mainRaster.diff \
-                  patch-CMakeLists_QGPGME.txt.diff
```

Comment:
Since patch-CMakeLists_QGPGME.txt.diff is no longer needed, the file should be removed too.

Reply:
I agree that the patch file should be removed and have done so now.  I wasn't sure if this needed to be done as it appears previous patch files still exist that aren't specifically called out by the Portfile.